modules/nf-core/kallistobustools/ref/tests/main.nf.test

```diff
@@ -0,0 +1,120 @@
+nextflow_process {
+
+    name "Test Process KALLISTOBUSTOOLS_REF"
+    script "../main.nf"
+    process "KALLISTOBUSTOOLS_REF"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "kallistobustools"
+    tag "kallistobustools/ref"
+
+    test("genome.fasta + genome.gtf + 'standard'") {
+
+        when {
+            process {
+                """
+                input[0] = file(params.modules_testdata_base_path + 'genomics/homo_sapiens/genome/genome.fasta', checkIfExists: true)
+                input[1] = file(params.modules_testdata_base_path + 'genomics/homo_sapiens/genome/genome.gtf', checkIfExists: true)
+                input[2] = "standard"
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(
+                    process.out.versions,
+                    process.out.t2g,
+                    process.out.cdna,
+                    process.out.intron,
+                    process.out.cdna_t2c,
+                    process.out.intron_t2c,
+                    ).match()
+                },
+                { assert file(process.out.index.get(0)).exists() }
```

Comment:
Cannot check for file content for these index files?

Reply:
I think we probably could, but I honestly didn't look into it too much haha. I think you're right so I'll see what I can do before merging

Reply:
Would make it slightly more robust. Checking for existence is my last resort when md5 or file content checks are impossible (e.g. a binary file with changing content due to timestamp or something).